kroxylicious-docs/docs/_assemblies/assembly-configuring-record-validation-filter.adoc

```diff
@@ -78,6 +78,48 @@ allowEmpty: true                                                 # <3>
 <2> if `true`, the validator allows keys and or values to be `null`. The default is `false`.
 <3> if `true`, the validator allows keys and or values to be empty. The default is `false`.
 
+.Example JWS Signature Validation Rule Definition
+
+The JWS Signature Validation rule validates that the configured Kafka record header contains a valid https://datatracker.ietf.org/doc/html/rfc7515[JSON Web Signature (JWS)] Signature.
+
+[source,yaml]
+----
+jwsSignatureValidationConfig:
```

Comment:
Do we have to have the `Config` in the name?

Reply:
I feel the same way about `schemaValidationConfig`. Taken to it's logical extreme _every_ property could be suffixed with `Config`. It adds nothing.